code/game/machinery/recharger.dm

```diff
@@ -1,22 +1,27 @@
+#define RECHARGER_POWER_USAGE_GUN 250
+#define RECHARGER_POWER_USAGE_MISC 200
```

Comment:
Please `#undef` these at the end of the file

Reply:
done